src/View3d.ts

```diff
@@ -146,6 +146,10 @@ export class View3d {
     return this.canvas3d.containerdiv;
   }
 
+  getCanvasDOMElement(): HTMLCanvasElement {
+    return this.canvas3d.renderer.domElement;
+  }
```

Comment:
Thinking about this and anticipating future things like exposing the Camera, maybe it would be better if view3d can just export the ThreeJsCanvas, and we tighten up the public interface on that...  to be considered in a future pr.